pkg/apis/compute/v1alpha1/config.go

```diff
@@ -1,99 +0,0 @@
-/*
-Copyright 2018 The Crossplane Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package v1alpha1
-
-import (
-	"fmt"
-
-	corev1alpha1 "github.com/crossplaneio/crossplane/pkg/apis/core/v1alpha1"
-	"github.com/ghodss/yaml"
-	kubectlv1 "k8s.io/client-go/tools/clientcmd/api/v1"
```

Comment:
Looks like the build is failing due to this import being removed from the code, but not updating the `Gopkg.lock` file.  I assume running `make vendor` will fix that.

From https://jenkinsci.upbound.io/blue/organizations/jenkins/crossplane%2Fbuild/detail/PR-328/3/pipeline:
```
# Gopkg.lock is out of sync:
k8s.io/client-go/tools/clientcmd/api/v1: in Gopkg.lock's input-imports, but neither imported nor required
03:46:58 [FAIL]
```

Reply:
Fixed.